src/client-functions/client-function-builder.js

```diff
@@ -12,7 +12,7 @@ import getCallsite from '../errors/get-callsite';
 const DEFAULT_EXECUTION_CALLSITE_NAME = '__$$clientFunction$$';
 
 export default class ClientFunctionBuilder {
-    constructor (fn, options, callsiteNames = {}) {
+    constructor (fn, options, callsiteNames = {}, clientFunctionWrongTypeMessage = MESSAGE.clientFunctionCodeIsNotAFunction) {
```

Comment:
Ok, then. But this shouldn't leak into Builder then. We need separate validation in `setNativeDialogHandler`